cmd/convertor/builder/fastoci_builder.go

```diff
@@ -187,6 +188,21 @@ func (e *fastOCIBuilderEngine) UploadImage(ctx context.Context) error {
 	return e.uploadManifestAndConfig(ctx)
 }
 
+// CheckForConvertedLayer TODO
```

Comment:
Comments to note this is not relevant for fastOCI because the final result from containerd is not reproducible..